docker/gemini.dockerfile

```diff
@@ -10,19 +10,24 @@ USER root
 RUN conda create --name py2 python=2.7 && \
     bash -c "source activate py2 && conda install --yes -c conda-forge -c bioconda gemini=0.30.2-0 vt=2015.11.10 variant-effect-predictor=87-0"
 
+
+RUN curl -s -o /usr/local/bin/pull-tutorial.sh https://raw.githubusercontent.com/statgenetics/statgen-courses/master/src/pull-tutorial.sh
+RUN chmod a+x /usr/local/bin/pull-tutorial.sh
+
+RUN sed -i '2 i \
+	pull-tutorial.sh gemini & \
+	'  /usr/local/bin/start-notebook.sh
+
+
 #Download datafiles 
 #Update the exercise handout
 
 USER jovyan
 
-RUN curl -fsSL http://statgen.us/files/gemini-tutorial-data.tar.bz2 -o gemini.tar.bz2 && tar jxvf gemini.tar.bz2 && rm -f gemini.tar.bz2
+RUN ( cd $HOME/work && curl -fsSL http://statgen.us/files/gemini-tutorial-data.tar.bz2 -o gemini.tar.bz2 && tar jxvf gemini.tar.bz2 && rm -f gemini.tar.bz2 )
```

Comment:
Here everything is downloaded to work directly. they will perhaps be lost once you mount the volumes? I think the right way is to download the HOME then either move or make ln, like annovar.

We should never ever download directly to `work`